docs/plugins/discovery-ec2.asciidoc

```diff
@@ -182,60 +229,138 @@ Management Console. It should look similar to this.
+Prefer the https://aws.amazon.com/amazon-linux-ami/[Amazon Linux AMIs] as these
+allow you to benefit from the lightweight nature, support, and EC2-specific
+performance enhancements that these images offer.
 
 ===== Networking
-* Networking throttling takes place on smaller instance types in both the form of https://lab.getbase.com/how-we-discovered-limitations-on-the-aws-tcp-stack/[bandwidth and number of connections]. Therefore if large number of connections are needed and networking is becoming a bottleneck, avoid https://aws.amazon.com/ec2/instance-types/[instance types] with networking labeled as `Moderate` or `Low`.
-* When running in multiple http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html[availability zones] be sure to leverage {ref}/allocation-awareness.html[shard allocation awareness] so that not all copies of shard data reside in the same availability zone.
-* Do not span a cluster across regions.  If necessary, use a cross cluster search.
 
-===== Misc
-* If you have split your nodes into roles, consider https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html[tagging the EC2 instances] by role to make it easier to filter and view your EC2 instances in the AWS console.
-* Consider https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/terminating-instances.html#Using_ChangingDisableAPITermination[enabling termination protection] for all of your instances to avoid accidentally terminating a node in the cluster and causing a potentially disruptive reallocation.
+* Smaller instance types have limited network performance, in terms of both
+https://lab.getbase.com/how-we-discovered-limitations-on-the-aws-tcp-stack/[bandwidth
+and number of connections]. If networking is a bottleneck, avoid
+https://aws.amazon.com/ec2/instance-types/[instance types] with networking
+labelled as `Moderate` or `Low`.
+
+* It is a good idea to distribute your nodes across multiple
+http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html[availability
+zones] and use {ref}/allocation-awareness.html[shard allocation awareness] to
+ensure that each shard has copies in more than one availability zone.
+
+* Do not span a cluster across regions. {es} expects that node-to-node
+connections within a cluster are reasonably reliable and offer high bandwidth
+and low latency, and these properties do not hold for connections between
+regions. Although an {es} cluster will behave correctly when node-to-node
+connections are unreliable or slow, it is not optimised for this case and its
+performance may suffer. If you wish to geographically distribute your data, you
+should provision multiple clusters and use features such as
+{ref}/modules-cross-cluster-search.html[cross-cluster search] and
+{stack-ov}/xpack-ccr.html[cross-cluster replication].
```

Comment:
I'm not sure how to correctly link to this page, but I see things like this elsewhere. @lcawl is this ok?